h2/src/main/org/h2/mvstore/MVStore.java

```diff
@@ -2696,12 +2696,12 @@ public void setReuseSpace(boolean reuseSpace) {
         this.reuseSpace = reuseSpace;
     }
 
-    public int getRetentionTime() {
-        return retentionTime;
+    public int getRetentionSpaceVersions() {
+        return retentionSpaceVersions;
     }
 
     /**
-     * How long to retain old, persisted chunks, in milliseconds. Chunks that
+     * How long to retain old, persisted chunks, in megabytes. Chunks that
```

Comment:
Megabytes?

The whole Javadoc of this method doesn't match with the new implementation.